bin/hostname/tests/hostname_test.sh

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2023 Klara, Inc.
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+#
+# These tests need to run in a multibyte locale with non-localized
+# error messages.
+#
+export LC_CTYPE=C.UTF-8
+export LC_MESSAGES=C
+
+root_dir='/usr/tests/bin/hostname'
+
+test_jail_conf='test-hostname-jail {
+    host.hostname = "test-hostname";
+    path = "/";
+    persist;
+}'
+
+init(){
+  echo "$test_jail_conf" > "$root_dir/jail.conf"
+  jail -f "$root_dir/jail.conf" -c test-hostname-jail
+}
+
+recycle(){
+  jail -f "$root_dir/jail.conf" -r test-hostname-jail
+  rm "$root_dir/jail.conf"
+}
+
+basic_head()
+{
+  atf_set "descr" "main test"
+}
+
```

Comment:
I usually see —as documented in `atf-sh(3)`— that `head`, `body`, and `cleanup` (optionally), don't get this space between subroutines.